elfo-core/src/message/any.rs

```diff
@@ -116,6 +125,24 @@ impl AnyMessage {
         M::_from_any_ref(self)
     }
 
+    /// Tries to downcast the message to a mutable reference to the concrete type.
+    ///
+    /// Note: it returns `Some(&mut self)` if `M` is [`AnyMessage`].
+    pub fn downcast_mut<M: Message>(&mut self) -> Option<&mut M> {
```

Comment:
`#[inline]`?

Reply:
Sure. While we’re at it, should `_unchecked()` versions also have `#[inline]`?

Reply:
Most likely it doesn't matter until it's public

Reply:
Okay. I added `#[inline]` to this one.